src/templates/tournaments/tournament_bracket.html

```diff
@@ -0,0 +1,184 @@
+{% extends "base.html" %}
+
+{% load static %}
+
+{% block content %}
+  <!DOCTYPE html>
+  <html lang="en">
+    <head>
+      <meta charset="UTF-8" />
+      <link rel="stylesheet" href="{% static 'css/tournament_bracket.css' %}" />
+      <link rel="stylesheet" href="{% static 'style.css' %}" />
+      <title>{{ tournament_name }} Bracket</title>
+    </head>
+    <body>
+      <div class="frame">
+        <div class="div">
+          <div class="group">
+            <div class="overlap-group">
+              <div class="rectangle"></div>
+              <div class="rectangle-2"></div>
+              <div class="text-wrapper">
+                {% if players.1 %}{{ players.1.username }}{% endif %}
+              </div>
+              <div class="text-wrapper-2">0</div>
```

Comment:
0 should not be hardcoded. Do something to get whether that player has won the match.